analyzers/tests/SonarAnalyzer.UnitTest/Common/AnalyzerLanguageTest.cs

```diff
@@ -34,8 +34,10 @@ public void ToString_ReturnsLanguageName()
         [DataRow("File.cs")]
         [DataRow("File.Cs")]
         [DataRow("File.CS")]
+        [DataRow("File.razor")]
         [DataRow(@"C:\Project\File.cs")]
         [DataRow(@"/c/Project/File.cs")]
+        [DataRow(@"/c/Project/File.razor")]
         public void FromPath_CS(string path) =>
```

Comment:
Nitpick: following the values on the previous DataRows for `.cs` files I would add some tests with uppercase and mixed .razor extension:
```cs
[DataRow("File.RAZOR")]
[DataRow("File.RAzor")]
```

Reply:
I though about this and to me they are superflu. The existing test cases already check the possible casing issues.